pkg/client/cmd/deploy.go

```diff
@@ -335,17 +335,19 @@ func deployList(cmd *cobra.Command, args []string) {
 	}
 
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"REVISION", "CURRENT", "AGE", "DESCRIPTION"})
+	table.SetHeader([]string{"REVISION", "AGE", "DESCRIPTION"})
 	table.SetRowLine(true)
 	table.SetAlignment(tablewriter.ALIGN_LEFT)
 	table.SetRowSeparator("-")
 	table.SetAutoWrapText(false)
 
-	sort.Sort(deploy.ByRevision(resp.Deploys))
+	sort.Sort(sort.Reverse(deploy.ByRevision(resp.Deploys)))
```

Comment:
why both `Sort` and `Reverse`?

Reply:
I guess it's easier to change `deploy.ByRevision` (and all the logic stays on the server side).

Reply:
The `Sort` only support sort in ascending order, the `Reverse` return the reverse order for data.

Reply:
gotcha:
```go
type reverse struct {
	// This embedded Interface permits Reverse to use the methods of
	// another Interface implementation.
	Interface
}
```
`sort/sort.go` in the stdlib...
sorry for the noise.